### ISSUE TITLE
Cannot use Captum for model interpratiblity

### ISSUE THREAD

**Issue by DhruvAwasthi** [qubvel/segmentation_models.pytorch#365]
I am using this library for semantic image segmentation task. Everything works good. Now after building and training the model, I am trying to use [Captum](https://captum.ai/) to understand on which features my model is attributing to. However I always end up with the error `RuntimeError: element 0 of tensors does not require grad and does not have a grad_fn`. 

I investigates into this, and the reason for this is when I predict the output using model that I have trained using this library, it output a tensor with `requires_grad=False`, while if we use pretrained models directly from torchvision,they output a tensor with `requires_grad=True`.

The tutorial I followed for model interpretability is [this](https://captum.ai/tutorials/Segmentation_Interpret).

**Comment by Nachimak28** [comment]
@DhruvAwasthi I tried investigating the issue, although not working on the captum notebook which you referred to, but just regarding the requires_grad value.

I tried it using the example notebook for camvid training as given in the example directory of the repo. 
As seen in the predict function definition in [this](https://github.com/qubvel/segmentation_models.pytorch/blob/master/segmentation_models_pytorch/base/model.py) file of the repo, all that the predict function does do:
`with torch.no_grad():
    x = self.forward(x)`

Hence the output tensor will have a `requires_grad=False`.

Instead if you try to pass the tensor directly through the model as follows:
`output=model(image_tensor)`
or 
`output=model.forward(image_tensor)`
and on printing the value of `output.requires_grad`, you would ideally get `True`

Hopefully this solves the issue
